ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml

```diff
@@ -89,7 +85,7 @@ objects:
 
     # Once the cluster is up, executes shared tests
     - name: test
-      image: registry.svc.ci.openshift.org/${NAMESPACE}/${TEST_STREAM}:${TEST_TAG}
+      image: registry.svc.ci.openshift.org/${NAMESPACE}/${TEST_IMAGESTREAM_TAG}
```

Comment:
this whole thing should be resolved locally with image lookup, no?

Reply:
hm?